x-pack/plugins/fleet/common/openapi/components/schemas/output.yaml

```diff
@@ -24,6 +24,15 @@ properties:
     type: object
   config_yaml:
     type: string
+  ssl:
+    type: object
+    properties: 
+      certificate:
+        type: string
+      certificate_authorities:
+        type: string
```

Comment:
this should be array type, right?
```
type: array
items:
  type: string
```

Reply:
Yes that's correct, I am going to change that